assets/css/settings-styles.css

```diff
@@ -270,7 +270,7 @@ body.woocommerce_page_wpo_wcpdf_options_page {
 }
 
 .nav-tab-wrapper a.nav-tab.nav-tab-active {
-	border-bottom: 3px solid #51266b;
+	border-bottom: 3px solid #6e1edc;
```

Comment:
Sorry, I meant to say the text color for the **Upgrade** tab, like this:

![Screenshot from 2023-04-12 13-32-07](https://user-images.githubusercontent.com/533273/231458012-3d181944-4741-41d1-8d2b-40bd36be403f.png)